src/main/java/jp/co/soramitsu/sora/didresolver/services/impl/VerifyServiceImpl.java

```diff
@@ -21,27 +27,44 @@
 @Slf4j
 public class VerifyServiceImpl implements VerifyService {
 
+  private static final ObjectMapper mapper = new ObjectMapper()
+      .registerModule(new JavaTimeModule())
+      .enable(SerializationFeature.INDENT_OUTPUT)
```

Comment:
use static import at this line and line below

Reply:
Sorry, how? 
Do you mean that I need to define this mapper somewhere and then import it?